uisdk/src/main/res/values-en-rGB/strings.xml

```diff
@@ -125,18 +124,10 @@
     <!-- FEEDBACK -->
     <string name="kh_uisdk_feedback_title">Please, tell us more about your trip</string>
     <string name="kh_uisdk_fees_and_taxes_included">Taxes and fees included</string>
-    <string name="kh_uisdk_filter">Filter</string>
```

Comment:
where these removed for the new quote list by any chance and were not moved to the feature branch?

Reply:
On develop we were having the translations which were related to the new quote list branch. I've removed them both from develop and master